features/create_an_article.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Create articles
+  As a journalist,
+  In order to write articles
+  I would like to be able to have an article creation form
+
+  Background:
+    Given I visit the create article page
```

Comment:
I want to see that the user is logged in as a journalist and can see a button that takes him to this `new_cms_art icle_path`. The button should only be visible to users that have the role of a journalist. 

The title of this feature is "Journalist can create article" and nowhere in the implementation code is the journalist mentioned